cli/cmd/root_test.go

```diff
@@ -242,6 +244,34 @@ func Test_runVersionCheck(t *testing.T) {
 			},
 			doNotWantOutput: "* Warning: Your Keptn CLI version (0.8.0) and Keptn cluster version (0.8.0) don't match.",
 		},
+		{
+			name:           "show version check if `AutomaticVersionCheck` is set to true",
+			cliVersion:     "0.8.0",
+			metadataStatus: http.StatusOK,
+			metadataResponse: keptnapimodels.Metadata{
+				Keptnversion: "0.8.1-dev",
+			},
+			wantOutput: "* Warning: Your Keptn CLI version (0.8.0) and Keptn cluster version (0.8.1-dev) don't match.",
+			cliConfig: func() config.CLIConfig {
```

Comment:
Doing this because changing `rootCLIConfig` directly can have impact on other tests.